added titles to the docs pages (#109)
dappforce/subsocial-docs: docs/develop/sdk/profiles/CreateProfiles.md

title: Create And Update Profiles
displayed_sidebar: developSidebar
---

<head>

>   <title>Create and Update Profile on the Subsocial Blockchain | JS SDK Guide</title>

Reviewer: ```suggestion
  <title>Creating and Updating Profiles With The Subsocial JS SDK</title>
```
